README.md

```diff
@@ -38,12 +38,12 @@ The main source files are:
 - `src/popup/popup.jsx` The popup window that shows when you press the extension icon
 - `src/background/background.js` Persists and facilitates planning
 - `src/contentScript/contentScript.js` Executes on web pages, executes actions and scrapes elements
-- `src/utils.js` Helper functions
+- `src/llm-utils.js` Helper functions
```

Comment:
should have a line for `utils.js` and `llm-utils.js` as both files are used now

Reply:
Yes, good catch, thanks. I've added one more line for LLM utils
``` `src/llm-utils.js` LLM helper functions```